src/IO/gpio/Output.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "config/config.h"
+#if USE_GPIO == 1

```

Comment:
Is it necessary to have `#if USE_GPIO == 1` on all files under `gpio`? I understand having the guard in other places such as the interface. Unless there are some compilation errors that may happen otherwise, couldn't we just remove them here? After all, they don't affect the functionality.

Reply:
Probably, but I think we are doing it like this right now for all our IO classes. I guess it technically makes the binary slightly smaller, but that doesn't really matter to us.